packages/datadog-plugin-graphql/src/utils.js

```diff
@@ -0,0 +1,40 @@
+function extractErrorIntoSpanEvent (config, span, exc) {
+  const attributes = {}
+
+  if (exc.name) {
+    attributes.type = exc.name
+  }
+
+  if (exc.stack) {
+    attributes.stacktrace = exc.stack
+  }
+
+  if (exc.locations) {
+    attributes.locations = []
+    for (const location of exc.locations) {
+      attributes.locations.push(`${location.line}:${location.column}`)
+    }
+  }
+
+  if (exc.path) {
+    attributes.path = exc.path
+  }
+
+  if (exc.message) {
+    attributes.message = exc.message
+  }
+
+  if (config.graphqlErrorExtensions) {
+    for (const ext of config.graphqlErrorExtensions) {
+      if (exc.extensions?.[ext]) {
+        attributes[`extensions.${ext}`] = exc.extensions[ext]
```

Comment:
`exc.extensions[ext]` needs to be stringified, was we want to limit the potential for unbounded nesting in user-provided extensions.